[2025-10-07T10:17:11.244+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T10:17:11.291+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:16:29.686945+00:00 map_index=10 [queued]>
[2025-10-07T10:17:11.320+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:16:29.686945+00:00 map_index=10 [queued]>
[2025-10-07T10:17:11.332+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-10-07T10:17:11.371+0000] {taskinstance.py:2890} INFO - Executing <Mapped(_PythonDecoratedOperator): process_batch> on 2025-10-07 10:16:29.686945+00:00
[2025-10-07T10:17:11.413+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=741) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T10:17:11.417+0000] {standard_task_runner.py:72} INFO - Started process 940 to run task
[2025-10-07T10:17:11.418+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'vnstock_weekly_etl_incremental', 'process_batch', 'manual__2025-10-07T10:16:29.686945+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmph0r7t5c1', '--map-index', '10']
[2025-10-07T10:17:11.421+0000] {standard_task_runner.py:105} INFO - Job 57: Subtask process_batch
[2025-10-07T10:17:11.488+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-07T10:17:11.635+0000] {task_command.py:467} INFO - Running <TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:16:29.686945+00:00 map_index=10 [running]> on host e3278e0f0bbc
[2025-10-07T10:17:12.594+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='vnstock_weekly_etl_incremental' AIRFLOW_CTX_TASK_ID='process_batch' AIRFLOW_CTX_EXECUTION_DATE='2025-10-07T10:16:29.686945+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-07T10:16:29.686945+00:00'
[2025-10-07T10:17:12.596+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-10-07T10:17:12.601+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SDG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.612+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SDD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.613+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SDC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.615+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SDB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.617+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SDA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.621+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SD9: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.621+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SD8: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.622+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SD7: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.623+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SD6: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.623+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SD5: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.624+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SD4: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.624+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SD3: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.629+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SD2: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.633+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SD1: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.634+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCY: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.634+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCV: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.635+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.636+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.637+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCO: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.638+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.641+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCJ: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.641+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.649+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.649+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.650+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.662+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SCA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.673+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SC5: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.680+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBV: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.683+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.684+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.685+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.686+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.687+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.687+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.688+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.688+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.689+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.690+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SBA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.691+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SB1: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.691+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SAV: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.701+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SAS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.701+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SAP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.702+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SAM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.703+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SAL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.709+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SAF: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.713+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SAC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.714+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý SAB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.715+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý S99: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.716+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý S96: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.717+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý S74: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:12.717+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T10:17:12.795+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-10-07T10:17:12.803+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=vnstock_weekly_etl_incremental, task_id=process_batch, run_id=manual__2025-10-07T10:16:29.686945+00:00, map_index=10, execution_date=20251007T101629, start_date=20251007T101711, end_date=20251007T101712
[2025-10-07T10:17:13.830+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T10:17:14.649+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-07T10:17:14.663+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
